ansible/roles/umount-volume/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- set_fact: ansible_python_interpreter=/usr/bin/python
+  when: (ansible_distribution == "CentOS" and ansible_distribution_major_version < '5')
+
+- name: if exists, remove additional mounts
```

Comment:
`if exists, remove additional mounts` change to `Unattached mount points that are passed in`, or something of that nature.